src/sage/graphs/graph_decompositions/tree_decomposition.pyx

```diff
@@ -852,6 +852,22 @@ def make_nice_tree_decomposition(graph, tree_decomp):
         sage: bip_one_four_TD = bip_one_four.treewidth(certificate=True)
         sage: make_nice_tree_decomposition(bip_one_four, bip_one_four_TD)
         Nice tree decomposition of Tree decomposition: Graph on 15 vertices
+
+    The following two examples are for testing GitHub Issue 36843::
```

Comment:
please use the following way to refer to the GitHub issue (this will generate a link):  

 :issue:`36843`

Reply:
You can use the form
```
Check that :issue:`36843` is fixed::
```